scriptmodules/emulators/jzintv.sh

```diff
@@ -27,6 +27,14 @@ function sources_jzintv() {
     # jzintv-YYYYMMDD/ --> jzintv/
     mv jzintv-[0-9]* jzintv
     cd jzintv/src
+    # archive files have CRLF linendings, patch has CR only
+    find . -type f \( -iname "*.c" -o -iname "*.h" \) -exec flip -u {} \+
```

Comment:
I'm not sure I like the converting of all the files to LF to apply the patch. Seems overkill. Also so you mean the patches has LF only not CR ?

Reply:
Well, yes. LF (0x0a) was ment.
Some other options I considered here: Change the source zip archive to LF (hosted at your site, must be remembered when a new source archive is deployed), convert patches to CRLF (awkward as RaspiOS uses UNIX style endings), use `--ignore-whitespace` in applyPatch (may have knock-on effects).

Reply:
I had no success when changing the patch line endings. `patch` hinted to use either `--ignore-whitespace` or `--binary`.
I changed it to `dos2unix` and limited the conversion to the source file which will be patched.